geode-docs/security/serialization.html.md.erb

```diff
@@ -0,0 +1,54 @@
+---
+title:  Serialization
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Apache Geode offers mechanisms to control and filter object serialization, particularly 
+ in the context of security and performance. This is primarily achieved through:
+
+
+##  Global Serialization Filter (Java 8)
```

Comment:
Is this for 'exactly Java 8' or 'at least Java 8'?

If it's for 'exactly Java 8', what should operators running on later versions do?

If it's 'at least Java 8', we can remove it, as Geode doesn't support older Java versions anyway.